@navikt/core/css/action-menu.css

```
@@ -0,0 +1,228 @@
+/* stylelint-disable csstools/value-no-unknown-custom-properties */
+/* stylelint-disable aksel/design-token-exists */
+.navds-action-menu__content {
+  --__ac-action-menu-content-p: 0.5rem;
```

Should we use token here?
```suggestion
  --__ac-action-menu-content-p: var(--a-spacing-2);
```